app/assets/stylesheets/public/_general2.scss

```diff
@@ -1,3 +1,5 @@
+// TODO: This is an old file and should probably be removed, with styles moved to appropriate places.
+
```

Comment:
The vast majority of this file is commented out. I figure it should get cleaned up during the public site work.